packages/avif/codec/pre.js

```diff
@@ -14,10 +14,6 @@ if (isRunningInCloudFlareWorkers || isRunningInNode) {
     };
   }
 
-  if (import.meta.url === undefined) {
-    import.meta.url = 'https://localhost';
-  }
```

Comment:
These may still be needed to solve import.meta.url issues with Cloudflare workers (I can check that later)